sycl/test-e2e/Basic/interop/Interop_level_zero_backend.cpp

```diff
@@ -0,0 +1,85 @@
+// RUN: %if any-device-is-level_zero %{ %{build} -isystem %sycl_include -o %t-l0.out %}
```

Comment:
Tests which do not execute anything, but only check compilation, should be placed under `sycl/test` and they should use `-fsyntax-only` flag as well.

I'm fine with old tests still staying incorrect and be fixed in separate PR, but new tests should be written in a right way.

An alternative is to actually run an executable you build here, because you seem to have a full test. In that case you don't need any `%if`. Just say `REQUIRES: level_zero`.

Also, I don't think that you need `-isystem %sycl_include` at all. `%{build}` automatically passes `-fsycl` and the latter sets up all necessary include paths automatically.

Reply:
It was removed because sycl/test exists: sycl/test/basic_tests/interop-level-zero-2020.cpp and sycl/test/basic_tests/interop-backend-traits-level-zero.cpp